src/components/LeftSidebar/ConversationsList/ConversationsList.vue

```diff
@@ -126,6 +126,9 @@ export default {
 // Override vue overflow rules for <ul> elements within app-navigation
 .conversations {
 	overflow: visible !important;
-	margin-top: 4px;
+	display: flex;
+	flex-direction: column;
+	gap: 4px;
```

Comment:
Variables here please :)
```suggestion
	gap: var(--default-grid-baseline, 4px);
```